### ISSUE TITLE
2.0.4: ImportError when running from commandline

### ISSUE THREAD

**Issue by badboiaustria** [RaiMan/SikuliX1#375]
Hi!

I am using Sikulix 2.0.4 on Windows 10 64bit, JAVA 11.

My script, which imports some own modules, runs fine on IDE. but when running from commandline it fails with the error:

`[error] ImportError ( No module named myglobals )`

this is the .bat I am calling to start from commandline:
```
cd C:\temp\SikuliX\Play.sikuli
java -jar C:\temp\SikuliX\sikulixide-2.0.4.jar -r C:\temp\SikuliX\Play.sikuli
```

this is the main script:
```
import org.sikuli.script.SikulixForJython # I tried this without success
from sikuli import *
import os
import os.path
import sys

myPath = os.path.dirname(getBundlePath()) # I tried this without success
if not myPath in sys.path:
    sys.path.append(myPath)

import myglobals
reload(myglobals)
``` 
this is myglobals (in the same directory as the main script):
```
-*- coding: utf-8 -*- # I need that, otherwise it wont import
from sikuli import *
import os
import os.path
import time
```
I also tried a workaround from bug #1504901 without success (very old version)
```
C:\temp\SikuliX\Play.sikuli>java -jar C:\temp\SikuliX\sikulixide-2.0.4.jar -r C:\temp\SikuliX\Play.sikuli
[error] script [ Play ] stopped with error in line 20
[error] ImportError ( No module named myglobals )
[error] --- Traceback --- error source first
line: module ( function ) statement
20: main ( <module> ) import myglobals
[error] --- Traceback --- end --------------
```
it still works fine from IDE, but not from commandline

Thanks in advance
Michael

**Comment by RaiMan** [comment]
a few remarks (valid for 2.0.4+):
- `import org.sikuli.script.SikulixForJython` is generally not needed, when running scripts in the SikuliX context (only when running with plain Jython)
- `from sikuli import *` is only needed in imported scripts, not in main scripts (internally done)
- `import sys, os, time` generally not needed (already imported by sikuli)
- scripts in the same folder as the main script are found by import without any additional measures
- `-- coding: utf-8 --` is needed in imported scripts, if they contain non-ascii characters (internally automatically done for main scripts)

So I setup your structure and had no problems:

a folder `C:\temp\SikuliX` containing:
```
myglobals.sikuli
Play.sikuli
sikulixide-2.0.4.jar
```
with Play.sikuli/Play.py:
```
import myglobals
```
and myglobals.sikuli/myglobals.py
```
from sikuli import *
popup("hello from myglobals")
```

Loading both in the IDE and running the main script, I get the popup.
Without modifying anything I do not get the popup anymore (which is correct without reload)

If I change something in myglobals and save it, I get the popup again running Play the next time (hence the changed script internally is auto-reloaded).

If I run the Play script from commandline as
`java -jar C:\temp\SikuliX\sikulixide-2.0.4.jar -r C:\temp\SikuliX\Play`
then I get the popup as expected.

**Comment by badboiaustria** [comment]
Hi!
Thanks to your help I found the solution:
My file (actually folder) myglobals.sikuli was located in the subdirectory of play.sikuli. That was working fine with the IDE.

From commandline I had to move the folder to the main sikulix directory.

Actually I do have a second file to import. It was also not found, and because of some reasons Sikulix generated no folder for that. However, moving it to the main directory made the trick.

Problem solved,
thanks
Michael
